docs/solutions/rhel74_image/variable.tf

```diff
@@ -22,25 +22,26 @@ variable "ipxe_instance" {
 	default = {
 		name = "ipxe-rhel74"
 		hostname = "ipxe-rhel74"
+		os = "Oracle Linux"
+		os-version = "7.4"
 		shape = "VM.Standard1.1"
 	}
 }
 
-variable "ipxe_instance_image_ocid" {
+variable "region_all_zeros_ocid" {
 	type = "map"
 	default = {
-		// Use the image "Oracle-Linux-7.4-2017.09.29-0"
-		us-phoenix-1 = "ocid1.image.oc1.phx.aaaaaaaa3g2xpzlbrrdknqcjtzv2tvxcofjc55vdcmpxdlbohmtt7encpana"
-		us-ashburn-1 = "ocid1.image.oc1.iad.aaaaaaaaawy2hh3nreaesyqcdp4m6csg4lwen6ya2njgiyjeu5sodiahlaxq"
-		eu-frankfurt-1 = "ocid1.image.oc1.eu-frankfurt1.aaaaaaaaydqkfzrcejyllfiwhcfqob2yyvkmytghwki6zcmhyciyruinokva"
+		us-phoenix-1 = "ocid1.image.oc1.phx.aaaaaaaay27pdopotkapf2ahjlsn2wxndui5hn5w37hd2wss4ses4ol5xs6a"
+		us-ashburn-1 = "ocid1.image.oc1.iad.aaaaaaaaqftkoa5web2r7w4ls3wekgqmqy5f7untloetfiozyqbv2ql6qidq"
+		eu-frankfurt-1 = "ocid1.image.oc1.eu-frankfurt-1.aaaaaaaah4rggbyglst25peqd7vnyjzl6n5lwogiyllb6jaircakom46nswq"
 	}
 }
 
-variable "region_all_zeros_ocid" {
+variable "ipxe_image_ocid" {
 	type = "map"
 	default = {
-		us-phoenix-1 = "ocid1.image.oc1.phx.aaaaaaaay27pdopotkapf2ahjlsn2wxndui5hn5w37hd2wss4ses4ol5xs6a"
-		us-ashburn-1 = "ocid1.image.oc1.iad.aaaaaaaaqftkoa5web2r7w4ls3wekgqmqy5f7untloetfiozyqbv2ql6qidq"
-		eu-frankfurt-1 = "ocid1.image.oc1.eu-frankfurt-1.aaaaaaaah4rggbyglst25peqd7vnyjzl6n5lwogiyllb6jaircakom46nswq"
+		us-phoenix-1 = "ocid1.image.oc1.phx.aaaaaaaaxklzl52nmabfp3466ilzfpo7lv737k44kih4jpo7nsmxjehwrdsq"
```

Comment:
Looks like this is image "Oracle-Linux-7.4-2018.01.10-0" - would be useful to mention that in a comment.